introduction_to_amazon_algorithms/linear_learner_mnist/linear_learner_mnist.ipynb

```diff
@@ -245,27 +249,25 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "import boto3\n",
     "import sagemaker\n",
     "\n",
     "sess = sagemaker.Session()\n",
     "\n",
-    "linear = sagemaker.estimator.Estimator(container,\n",
-    "                                       role, \n",
-    "                                       train_instance_count=1, \n",
-    "                                       train_instance_type='ml.c4.xlarge',\n",
-    "                                       output_path=output_location,\n",
-    "                                       sagemaker_session=sess)\n",
-    "linear.set_hyperparameters(feature_dim=784,\n",
-    "                           predictor_type='binary_classifier',\n",
-    "                           mini_batch_size=200)\n",
-    "\n",
-    "linear.fit({'train': s3_train_data})"
+    "linear = sagemaker.estimator.Estimator(\n",
+    "    container,\n",
+    "    role,\n",
+    "    train_instance_count=1,\n",
```

Comment:
not v2 params... should be instance_count
```suggestion
    "    instance_count=1,\n",
```

Reply:
Both train_instance_type and train_instance_count are supported in v2 as well. Though, as per your suggestion, I changed them to instance_type and instance_count in all the notebooks.

Let's do other such changes after re:invent. Right now, priority is functioning notebooks.